ansible/monitoring.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright StackHPC, 2017
+#
+---
+- name: Configure instance monitoring
+  hosts: cluster
+  become: yes
+
+  roles:
+  - role: stackhpc.monasca-agent
+    keystone_url: "http://{{ controller_vip }}:5000/v3"
+    monasca_user: root
+    monasca_agent_user: "{{ monasca_agent_p3_username }}"
+    monasca_agent_password: "{{ monasca_agent_p3_password }}"
+    monasca_agent_project: "{{ project_name }}"
+    monasca_agent_version: 2.2.0
+    monasca_endpoint_type: public
+    monasca_project_domain_name: Default
+    monasca_user_domain_name: Default
+    monasca_region_name: RegionOne
+    monasca_service_type: monitoring
+    monasca_log_level: DEBUG
+    monasca_agent_custom_plugin_repos:
+    - "https://github.com/stackhpc/stackhpc-monasca-agent-plugins.git@master"
```

Comment:
How would I configure the plugins used by the agent? Using the `monasca_agent_plugin` module in another playbook?

Reply:
Good question. That's a possibility. None of these plugins need configuring so I haven't needed to so far. When the time comes I can address this under another PR.